Docs/source/ML_Tutorial.rst

```diff
@@ -4,13 +4,32 @@
 .. role:: fortran(code)
    :language: fortran
 
-Tutorials/ML
+ML/PYTORCH
 ==========================
 
-In ML/PYTORCH there is an example of using a pre-trained PyTorch model in the
-AMReX framework.  In this example we initialize data on a MultiFab, copy
-the data into a PyTorch tensor, call the model, and load the result back
-into a MultiFab.  The example works on both the host and GPU.
-A model is included as an example (``/Exec/model.pt``).
-See ``ML/PYTORCH/README.md`` for ``libtorch`` download instructions.
+In ``amrex-tutorials/ExampleCodes/ML/PYTORCH`` there is an example demonstrating the usage of a pre-trained PyTorch model in the AMReX framework.  In this example, we initialize data on a MultiFab, copy the data into a PyTorch tensor, call the model, and load the result back into a MultiFab.  The program works on both the host (CPU) and GPU.
+
+**Running AMReX application with PyTorch model**
+------------------------------------------------
+
+Below is a step-by-step guide to succesfully run an AMReX program that uses PyTorch model. It is based on ``ML/PYTORCH/README.md`` and will require the model to have been saved as TorchScript. For more information on TorchScript, please visit `here <https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html>`.
```

Comment:
I'm a little confused by "save as TorchScript", is it "save as a TorchScript" or does the filename have to be `TorchScript`?

Reply:
It should be a TorchScript file, which in this example is name `model.pt`. I should probably make that clearer in the next section.